src/modules/launcher/Plugins/Microsoft.Plugin.Program/Views/ProgramSetting.xaml.cs

```diff
@@ -51,7 +51,6 @@ private void ReIndexing()
             Task.Run(() =>
             {
                 Dispatcher.Invoke(() => { indexingPanel.Visibility = Visibility.Visible; });
-                Main.IndexPrograms();
                 Dispatcher.Invoke(() => { indexingPanel.Visibility = Visibility.Hidden; });
             });
         }
```

Comment:
Can we remove the task.Run then?

Reply:
We can remove this whole file!  We need to design the settings for Run, but I don't think we want reindexing to be a manual process.   I just removed this to get it to compile as it's no longer a static method.